www/js/survey/enketo/enketo-add-note-button.js

```diff
@@ -70,8 +70,6 @@ angular.module('emission.survey.enketo.add-note-button',
     if (momentStop) {
       prefills.End_date = momentStop.format('YYYY-MM-DD');
       prefills.End_time = momentStop.format('HH:mm:ss.SSSZ');
-    } else {
-      prefills.End_date = momentBegin.format('YYYY-MM-DD');
```

Comment:
@JGreenlee I am not sure this is the fix. If we don't have a defined end date, which version of the survey will be displayed? Can you add a screenshot of what it would like like?

Reply:
There is only one version of the survey, but redundant fields are hidden.
If the Start and End Date are both supplied and they are the same, then we they don't show up.

The problem was that I was filling an End Date, even when there was no End Time. So both Date fields would be hidden.

Here's how it now shows on the last place, where there is no exit_ts

<img src=https://github.com/e-mission/e-mission-phone/assets/15843932/591004a6-3bd0-42f6-b561-ae72cfdfb13c width=300>

Clicking the empty End Date field allows you to pick, initializing on 'today'